src/sections/ProjectCards.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { navigate } from 'gatsby';
+import ProjectCard from '../components/ProjectCard';
+
+const ProjectCards = ({ projects }) => (
+    <div className="flex flex-col">
+        {projects.map((project) => (
+            <ProjectCard
+                key={project.id}
+                onClick={() => { navigate(`/${project.slug}`); }}
```

Comment:
use <Link to={`/${project.slug}`}> instead of onClick

Reply:
Link used